### ISSUE TITLE
How to use as a library

### ISSUE THREAD

**Issue by tbozinis** [Cadene/block.bootstrap.pytorch#6]
Hello @Cadene , I have a question similar to issue #1.
My end goal is to use your work as a library and build a VQANet with the block.yaml optionsin order to use the model for VQA. The dataset I want to use is vqa2. The ideal scenario is to use pre trained models (both for VQA and the image features extraction), but I could train it my self if I must (the VQA one).

#### So my questions are:
1. Is there a pre trained model that I could use for VQA (and load it on a VQANet)?
2. What **model** did you use for the image **features extraction**? Where can I find a pre trained features extractor and how to use it to feed images to the VQANet directly? (Both for making the .pth file you use for training and on making the tensor to use on the VQANet model)

> Also if there is a wiki page or notes on how to do any of the things I asked, could you point me to that? It will certainly help in the future...
<br>Thank you for your time !

**Comment by tbozinis** [comment]
Also, can I and if so how to use [faster rcnn](https://github.com/Cadene/pretrained-models.pytorch/blob/master/pretrainedmodels/models/cafferesnet.py) for features extraction?

**Comment by Cadene** [comment]
@tbozinis Thank you for your interest :)

You can download the visual features like [this](https://github.com/Cadene/block.bootstrap.pytorch#3-download-datasets).

> 1. Is there a pre trained model that I could use for VQA (and load it on a VQANet)?

We uploaded several pretrained models that you can download like [this](https://github.com/Cadene/block.bootstrap.pytorch#vqa2)

Then you can evaluate a pretrained model on the test set another time like this:
```
python -m bootstrap.run \
-o logs/vqa2/block/options.yaml \
--exp.resume best_eval_epoch.accuracy_top1 \
--dataset.train_split \
--dataset.eval_split test \
--misc.logs_name test_2
```

> 2. What model did you use for the image features extraction?

We used the pretrained FasterRCNN from the [bottom-up github](https://github.com/peteanderson80/bottom-up-attention). We had to install Caffe.

Best luck!

**Comment by tbozinis** [comment]
Great, thank you for the answer. I will try them out!
